src/js/utils/backbone.events.ts

```diff
@@ -11,9 +11,38 @@
 
 // Mixin module modified into a class which can be extended
 
-const slice = [].slice;
+type EventsAction = 'on'|'once'|'off'|'trigger';
 
-export default class Events {
+type EventListenerDictionary = {
+    [name: string]: EventListener;
```

Comment:
`name: EventsAction`?

Reply:
No, this is the map of listeners - ex: `"ready": onReady`.